webapp/src/main/java/org/opentestsystem/rdw/admin/service/GroupService.java

```diff
@@ -8,9 +8,27 @@
 
 import java.util.Set;
 
-@PreAuthorize("hasAuthority('PERM_GROUP_READ')")
+@PreAuthorize("hasAuthority('PERM_GROUP_WRITE')")
```

Comment:
Could this admin portal have other functions besides just group at some point?

Also I'd still like some basic endpoints such as user and translate to flow through so we can at least render an error page to the user should they not have access.